openduck-py/openduck_py/routers/voice.py

```diff
@@ -412,3 +423,59 @@ async def audio_response(
     await responder.response_queue.join()
     await websocket.close()
     await log_event(db, session_id, "ended_session")
+
+
+async def connect_daily():
+    session_id = str(uuid4())
+
+    Daily.init()
+    mic = Daily.create_microphone_device(
+        "my-mic", sample_rate=OUTPUT_SAMPLE_RATE, channels=1, non_blocking=True
+    )
+    speaker = Daily.create_speaker_device(
+        "my-speaker", sample_rate=WS_SAMPLE_RATE, channels=1
+    )
+    Daily.select_speaker_device("my-speaker")
```

Comment:
does this string make a difference?

Reply:
I think it could be anything but it's like a variable name for the speaker